sql/connect/common/src/main/protobuf/spark/connect/ml.proto

```diff
@@ -51,7 +51,7 @@ message MlCommand {
   // Command to delete the cached object which could be a model
   // or summary evaluated by a model
   message Delete {
-    ObjectRef obj_ref = 1;
+    repeated ObjectRef obj_refs = 1;
```

Comment:
nit. Could you update the above comment consistently, @zhengruifeng ?
```
- // Command to delete the cached object 
+ // Command to delete the cached objects 
```

Reply:
sure